orca-sql/src/test/groovy/com/netflix/spinnaker/orca/sql/pipeline/persistence/SqlExecutionRepositorySpec.groovy

```diff
@@ -501,4 +503,42 @@ class SqlExecutionRepositorySpec extends ExecutionRepositoryTck<SqlExecutionRepo
       new ExecutionCriteria().setPageSize(1).setSortType(BUILD_TIME_ASC)
     ).size() == 0
   }
+
+  @Unroll
+  def "can retrieve #type by status and time window"() {
```

Comment:
i wanted to add a unit test for the other function (`retrieveExecutionsWithSpecificStageTypesInTimeWindow`) but `H2` doesn't support `JSON_EXTRACT` so it wasn't working. Let me know if there a way to make it work